src/components/table/__spec__.js

```diff
@@ -931,6 +930,21 @@ describe('Table', () => {
       expect(parent.className).toEqual('carbon-table foo');
     });
 
+    it('renders a caption tag when a caption prop is given', () => {
+      let wrapper = render(
```

Comment:
Because I'm only interested in verifying that a `caption` is included in the rendered HTML.